### PR TITLE
feat: [CDS-45863]: add enable, disable icons

### PR DESCRIPTION
**Pull request by udayvunnam**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/disable.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg fill="none" height="16" viewBox="0 0 17 16" width="17" xmlns="http://www.w3.org/2000/svg">
```
Comment (AshwinBhatkal-Harness) [inline packages/icons/src/disable.svg:1]: Is this a toggle?

**Review by ishantmehtaharness — DISMISSED** [review]
Looks Good Uday

File: packages/icons/src/disable.svg (hunk 1 above)
Comment (AshwinBhatkal-Harness) [reply packages/icons/src/disable.svg:1]: I see height here. Is it required?
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

Comment (udayvunnam) [reply packages/icons/src/disable.svg:1]: yes

Comment (AshwinBhatkal-Harness) [reply packages/icons/src/disable.svg:1]: Shouldn't toggle be interactive rather than icon?

Comment (udayvunnam) [reply packages/icons/src/disable.svg]: good callout. Updated!

Comment (udayvunnam) [reply packages/icons/src/disable.svg]: image


It's needed here

Comment (udayvunnam) [reply packages/icons/src/disable.svg]: image

Comment (AshwinBhatkal-Harness) [reply packages/icons/src/disable.svg]: Got it. Thank you

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**